### PR TITLE
Add ci cd using travis

### PR DESCRIPTION
**Pull request by mkdesignn**
- Implement and setup travis-ci pipeline
- Fix the login test

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +95,16 @@
                 <groupId>org.codehaus.gmavenplus</groupId>
                 <artifactId>gmavenplus-plugin</artifactId>
                 <version>1.6</version>
+                <configuration>
+                    <testSources>
+                        <testSource>
+                            <directory>src/test/groovy</directory>
```
Comment (Imany-apk) [inline pom.xml]: Is it true that test codes is under `src/test/groovy/java`, now?
Shouldn't they be under `src/test/groovy`? 
`java` folder is my question actually

File: src/main/java/com/example/jwt/entity/User.java [hunk 2]
```diff
@@ -18,6 +16,7 @@
 @Getter
 @Builder
 @Table(name = "users")
+@AllArgsConstructor
 public class User implements Serializable {
```
Comment (Imany-apk) [inline src/main/java/com/example/jwt/entity/User.java]: this class was renamed to `AppUser`
what's happened?

File: src/main/java/com/example/jwt/transformer/BaseResponseDTO.java [hunk 3]
```diff
@@ -1,7 +1,11 @@
 package com.example.jwt.transformer;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import lombok.Data;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"data"})
```
Comment (Imany-apk) [inline src/main/java/com/example/jwt/transformer/BaseResponseDTO.java]: Is it for response JSON ordering?

File: src/main/resources/db/migration/V1__init.sql [hunk 4]
```diff
@@ -1,5 +1,5 @@
 create table users(
-    id bigint primary key unique auto_increment,
+    id bigint auto_increment,
```
Comment (Imany-apk) [inline src/main/resources/db/migration/V1__init.sql]: why remove them?

Comment (mkdesignn) [reply src/main/resources/db/migration/V1__init.sql]: When we use **id bigint primary key unique auto_increment** it's not going to work H2.

File: src/main/java/com/example/jwt/entity/User.java (hunk 2 above)
Comment (mkdesignn) [reply src/main/java/com/example/jwt/entity/User.java]: Nope, I have changed it back to user.

File: src/main/java/com/example/jwt/transformer/BaseResponseDTO.java (hunk 3 above)
Comment (mkdesignn) [reply src/main/java/com/example/jwt/transformer/BaseResponseDTO.java]: I have removed them.

File: pom.xml (hunk 1 above)
Comment (mkdesignn) [reply pom.xml]: I have changed the directoy.

File: lombok.config [hunk 5]
```diff
@@ -0,0 +1 @@
+lombok.anyConstructor.addConstructorProperties=true
```
[the remark this answers is not in the document]
Comment (mkdesignn) [reply lombok.config]: I have removed the file, We don't need it.

**Pull request merged**